dockerfiles/ubuntu-trusty-all/Dockerfile

```diff
@@ -65,14 +65,15 @@ RUN curl -sL https://github.com/stedolan/jq/releases/download/jq-1.5/jq-linux64
          -o /usr/bin/jq \
```

Comment:
I thought you said Trusty was EoL'd? In that case, can't you just `git rm` this whole subdirectory?